<?xml version="1.0"?>

<launch>

    <arg name="veh" default="jackal" />
    <arg name="goal_dis" default="1.5" />
    <arg name="mission" default="garage" />
    <arg name="frame" default="map" />
    <arg name="linear_scale" default="0.3" />
    <arg name="angular_scale" default="0.3" />
    <arg name="scan_in" default="RL/scan" />
    <!-- map truth_map_posestamped -->
    <group ns="$(arg veh)">

        <node pkg="subt_rl" type="goal_nav_tf_vel_jackal.py" name="goal_nav_jackal_vel_rl" output="screen">
            <param name="frame" value="$(arg frame)" />
            <param name="goal_dis" value="$(arg goal_dis)" />
            <param name="mission" value="$(arg mission)" />
            <param name="linear_scale" value="$(arg linear_scale)" />
            <param name="angular_scale" value="$(arg angular_scale)" />
            <remap from="goal_in" to="/move_base_simple/goal" />
            <remap from="laser_in" to="/$(arg scan_in)" />
            <remap from="odom_in" to="/truth_map_posestamped" />
            <remap from="joy" to="/joy" />
            <remap from="cmd_out" to="/X2/cmd_vel" />

        </node>

    </group>


</launch>